src/app/InteractionModelEngine.cpp

```diff
@@ -381,14 +454,33 @@ Protocols::InteractionModel::Status InteractionModelEngine::OnReadInitialRequest
             {
                 TLV::TLVReader pathReader;
                 eventpathListParser.GetReader(&pathReader);
-                err = TLV::Utilities::Count(pathReader, requestedEventPathCount, false);
+                TLV::Utilities::Count(pathReader, requestedEventPathCount, false);
```

Comment:
Why is this no longer checking the returned CHIP_ERROR?  It should be checking it, I would think.

Reply:
agree, updated, thanks

Reply:
updated with                 ReturnErrorCodeIf(TLV::Utilities::Count(pathReader, requestedEventPathCount, false) != CHIP_NO_ERROR,   Status::InvalidAction);